Cargo.toml

```diff
@@ -11,11 +11,11 @@ edition = "2021"
 
 [dependencies]
 serde = { version = "1", default-features = false }
-digest = { version = "0.10", default-features = false }
+digest = { version = "0.11.0-pre.9", default-features = false }
 
 [dev-dependencies]
-sha2 = "0.10"
-k256 = { version = "0.13", features = ["serde", "pem"] }
+sha2 = "0.11.0-pre.3"
+k256 = {git = "https://github.com/RustCrypto/elliptic-curves", features = ["serde", "pem"]}
```

Comment:
I'm not sure about this one, does `0.14.0-pre.0` not work?

Reply:
No it doesn’t. Well, I’m sure it works here in this crate, but elsewhere e.g. in synedrion it does not. It’s two months old I think so it needs an update. :/

I’m fine using this branch if we want to wait for a new k256 release.

Reply:
Yeah, there was a recent commit there that bumped the dependencies again. I think it all hinges on `crypto-bigint` being a pre-release.